roles/bootstrap-os/tasks/bootstrap-debian.yml

```diff
@@ -13,6 +13,36 @@
   environment: {}
   tags: facts
 
+- name: Check http::proxy in /etc/apt/apt.conf
+  raw: grep -qs 'Acquire::http::proxy' /etc/apt/apt.conf
```

Comment:
"proxy" can be capitalized according to https://wiki.debian.org/AptConf

Reply:
The Debian distribution has conflicting information
https://debian-handbook.info/browse/en-US/stable/sect.apt-get.html

```
If the Web can only be accessed through a proxy, add a line like Acquire::http::proxy "http://yourproxy:3128". 
```

Are you get error when install kubespray?

Reply:
No, currently I'm looking through the roles here to assess their quality and look for potential error sources before running them.

I guess in this case it might even be necessary to check the source code of `apt` to get a definite answer. Anyways, I guess an additional `i` in the grep invocation won't really hurt.

Reply:
Added i to grep.
Change 'Acquire::http::proxy' to 'Acquire::http::Proxy'
https://github.com/kubernetes-sigs/kubespray/pull/3963/files